src/styles/blocks/_expertise.scss

```diff
@@ -0,0 +1,118 @@
+.expertise {
```

Comment:
As it was mentioned in the checklist (16. In the Testimonials and Our expertise section add hover effects for each review. ) You can add hover effects for expertise section too.